simple_encryption_algo_python3/decrypt.py

```diff
@@ -0,0 +1,11 @@
+f1=open("encrypted_file.txt","r")
```

Comment:
move to encryption_algorithms directory, rename file or create a sub directory inside, with proper documentation.